### DISCUSSION TITLE
Allow font changes

### DISCUSSION THREAD

**Discussion by LightTreasure** [labithiotis/zed-themes#85] [Ideas]
This is an awesome tool! 

I think it could benefit greatly from having a font selector on the themes list page as well as the themes editor page. Reasons include:

- A particular font can affect how a theme looks, and vice versa
- Users looking to try out themes might feel differently about it when using their font of choice
- Theme creators will be able to see the effect of using different fonts with their themes
- This website could then also be used to test out different fonts!

**Comment by labithiotis** [comment]
@LightTreasure Thanks for the suggestion, this was something I had considered, although allowing custom fonts will be tricker to implement as we will need to add some infra for upload/host fonts. I wonder if the first step should be offering any Google Fonts (a limit set of only [monospace](https://fonts.google.com/?classification=Monospace), maybe??).

**Reply by LightTreasure** [reply]
That is a good starting point! Another good starting point would be to support the popular [Nerd Fonts](https://www.nerdfonts.com/)!
